test/input_gen/genInput.py

```diff
@@ -51,6 +51,12 @@ def save(filename, *data):
 def gen_tensor(shape, dtype=dtypes.float32):
     return np.random.randint(1, 10, shape).astype(np.float32)
 
+##
+# @brief generate random tensor
+def gen_tensor_int(shape):
+  rand_t = np.random.randint(1, 100, size=shape)
```

Comment:
please limit max value to 10. 
Values to 100 causes overflow with bigger layers/values.

Reply:
Also, similar values can bring out errors like from pooling etc.